vnext/JSI.Desktop.UnitTests/JSI.Desktop.UnitTests.vcxproj

```diff
@@ -53,16 +46,16 @@
         _WIN32_WINNT=$(WinVer);
         WIN32;
         _WINDOWS;
-        REACTNATIVEWIN32_EXPORTS;
         FOLLY_NO_CONFIG;
         NOMINMAX;
-        GLOG_NO_ABBREVIATED_SEVERITIES;
         _HAS_AUTO_PTR_ETC;
         CHAKRACORE;
         RN_EXPORT=;
         GTEST_LANG_CXX11;
         %(PreprocessorDefinitions)
       </PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(USE_HERMES)'=='true'">USE_HERMES;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
is this not being added by react.cpp.props?

Reply:
No, perhaps it should be there. We were depending on a side-effect of including Shared.vcxitems which doesn't need to be here.

---
In reply to: [422409109](https://github.com/microsoft/react-native-windows/pull/4840#discussion_r422409109) [](ancestors = 422409109)

Reply:
aha! yes, it should - really in a react.cpp.targets that gets included after everything, but putting it in react.cpp.props is fine for now